### PR TITLE
T2276

### PR DESCRIPTION
**Pull request by cadecairos**
Lighthouse ticket: https://webmademovies.lighthouseapp.com/projects/65733/tickets/2276-limit-amount-of-fetched-tweets-to-something-sane

### CODE REVIEW

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_check** from @mjschranz received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/60b04c09f80af72/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @mjschranz received. Current queue size: 1

Live output at: http://botio.butyr.org:8000/ca66e888545d01a/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Failed

Full output at http://botio.butyr.org:8000/60b04c09f80af72/output.txt

Total script time: 1.73 mins

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/ca66e888545d01a/output.txt

Total script time: 2.11 mins

Server is running on http://pr995.butyr.org/templates/basic/
Run the Test Suite: http://pr995.butyr.org/test/

File: templates/assets/editors/twitter/twitter-editor.js [hunk 1]
```diff
@@ -47,6 +48,14 @@
      * @param {Object} updateOptions: TrackEvent properties to update
      */
     function updateTrackEventWithoutTryCatch( trackEvent, updateOptions ) {
+      if ( updateOptions.numberOfTweets ) {
+        if ( updateOptions.numberOfTweets > _maxTweets ) {
+          updateOptions.numberOfTweets = _maxTweets;
+          setErrorState( "The maximum number of tweets you may retrieve is " + _maxTweets );
```
Comment (secretrobotron) [inline templates/assets/editors/twitter/twitter-editor.js]: Add a `.` at the end of the sentence. :)

**Pull request merged**